### PR TITLE
[Add] troubleshooting guide / keep getting the not connected to wi-fi or vpn error

### PR DESCRIPTION
**Pull request by neoarz**

### CODE REVIEW

File: docs/troubleshooting/troubleshooting.mdx [hunk 1]
```diff
@@ -62,6 +64,17 @@ Another common issue during sign-in is not receiving a verification code when us
    - Use this code in Sidestore to complete the sign-in process.
 
 
+## Can't Refresh or Install Apps
+
+#### Resolving No Wi-Fi or VPN Error
+
+1. Disable your cellular data / connect to a stable Wi-Fi network.
+2. Make sure Wi-Fi is not restricted (such as school Wi-Fi).
```
Comment (polymo1) [inline docs/troubleshooting/troubleshooting.mdx:72]: maybe something closer to:

"Ensure that Apple domains, and the domain for your anisette server, are not restricted (e.g. school/work Wi-Fi)"

File: docs/troubleshooting/troubleshooting.mdx [hunk 2]
```diff
@@ -62,6 +64,17 @@ Another common issue during sign-in is not receiving a verification code when us
...
+3. Verify VPN is connected in the WireGuard app.
+4. Turn off, turn on WireGuard, and wait a few seconds in SideStore before trying to refresh.
```
Comment (polymo1) [inline docs/troubleshooting/troubleshooting.mdx:74]:
```
- Turn off, turn on
+ Turn off, then turn back on
```

File: docs/troubleshooting/troubleshooting.mdx [hunk 3]
```diff
@@ -62,6 +64,17 @@ Another common issue during sign-in is not receiving a verification code when us
...
+5. Create a brand new pairing file.
+   - If none of the above worked it is very likely that the pairing file is corrupted. You can reference the documentation on how to create a new pairing file [here](https://docs.sidestore.io/docs/getting-started/pairing-file/). 
+   - After creating a new pairing file, go to sidestore settings and press "Reset pairing file", then choose the new pairing file you just created.
```
Comment (polymo1) [inline docs/troubleshooting/troubleshooting.mdx:77]:
```
- sidestore
+ SideStore
```

**Review by polymo1 — CHANGES_REQUESTED** [review]

File: docs/troubleshooting/troubleshooting.mdx (hunk 3 above)
Comment (neoarz) [reply docs/troubleshooting/troubleshooting.mdx]: fixed

**Comment by polymo1** [comment]
lgtm

**Pull request merged**